### PR TITLE
Fix json format

### PR DESCRIPTION
**Pull request by stuartleeks**
--------------------MESSAGE FROM ADMIN, DELETE BEFORE SUBMITTING----------------------

Thank you for deciding to contribute to this repository, we're thrilled to have you contribute a template or a fix! :tada: Before we proceed, please be sure to review the contribution guidelines that will make the review process go smoother.
https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md

We've created an outline of recommended sections to fill out that will help make this Pull Request awesome!

--------------------MESSAGE FROM ADMIN, DELETE BEFORE SUBMITTING----------------------

### Changelog
*
*
*
*
*

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@stuartleeks__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (stuartle). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: 101-acs-swarm/azuredeploy.parameters.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
       "value": "SwarmPreview"
     },
     "masterCount": {
-      "value": "3"
+      "value": 3
```
Comment (rgardler) [inline 101-acs-swarm/azuredeploy.parameters.json]: This breaks validation, please see the CI output.

File: 101-acs-swarm/azuredeploy.parameters.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
       "value": "GEN-UNIQUE"
     },
     "agentCount": {
-      "value": "1"
+      "value": 1
```
Comment (rgardler) [inline 101-acs-swarm/azuredeploy.parameters.json]: This breaks validation, please see the CI output.

File: 101-acs-swarm/azuredeploy.parameters.json (hunk 1 above)
Comment (rgardler) [inline 101-acs-swarm/azuredeploy.parameters.json]: This breaks validation, please see the CI output.

File: 101-acs-swarm/azuredeploy.parameters.json (hunk 2 above)
Comment (singhkay) [inline 101-acs-swarm/azuredeploy.parameters.json]: @rgardler, actually this is fine. The metdata file is invalid JSON. @stuartleeks Can you check?

Comment (stuartleeks) [inline 101-acs-swarm/azuredeploy.parameters.json]: I haven't changed anything in the metadata file, so I'm not sure that my change is the issue here ;-)

I've taken a look at the metadata and it looks like there's a newline issue (maybe from pasting content). I've pushed another update that will hopefully fix that.

File: 101-acs-swarm/azuredeploy.parameters.json (hunk 1 above)
Comment (stuartleeks) [inline 101-acs-swarm/azuredeploy.parameters.json]: The CI output error is for another error (in metadata as per previous line comment).

**Comment by singhkay** [comment]
CI passed, merged

File: 101-acs-swarm/azuredeploy.parameters.json (hunk 2 above)
Comment (rgardler) [inline 101-acs-swarm/azuredeploy.parameters.json]: Apologies for the noise. Thank you for finding the real problem elsewhere.

**Pull request merged**